core/spec.md

```diff
@@ -436,6 +440,117 @@ attributes. However they MUST adhere to the following rules:
   example, use of a model (or domain) specific prefix could be used to help
   avoid possible future conflicts.
 
+#### `xid`
+
+##### Referencing xRegistry Entities
+
+An `xid` is a URI according to
+[RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986) that
+uniquely identifies an entity within an xRegistry. Its purpose is to
```

Comment:
s/an Registry/a Registry/

Also, is it just "within a Registry" or is it mean to be more globally identifying? I thought it was for cross-registry identification too